microsoft-edge/webview2/reference/javascript/sharedbufferreceivedevent.yml

```diff
@@ -0,0 +1,131 @@
+### YamlMime:TSType
+name: SharedBufferReceivedEvent
+uid: 'WebView2Script!SharedBufferReceivedEvent:class'
+package: WebView2Script!
+fullName: SharedBufferReceivedEvent
+summary: >-
+  Event object for the `chrome.webview.sharedbufferreceived` event. This event is dispatched when
+  `CoreWebView2.PostSharedBufferToScript` is successfully called.
+remarks: |-
+
+
+  #### Examples

```

Comment:
```suggestion
```
Deleting the Examples heading might be a solution to a comment that reads:

"Can we delete the H2 called Remarks and then promote Examples to H2? The double heading without a sentence in between looks incomplete."

Also, if it's a single example, 's' is not needed.

Reply:
The example header produced by [MSDN .NET toolchain](https://review.learn.microsoft.com/en-us/help/onboard/admin/reference/dotnet/?branch=main) i.e. those on [.NET API browser](https://learn.microsoft.com/en-us/dotnet/api/) uses plural `Examples` regardless of number of examples - is it okay to just use `Examples` for consistency?